praw/__init__.py

```diff
@@ -380,9 +389,12 @@ def handle_redirect():
                 request.url = url
                 kwargs['_cache_key'] = (normalize_url(request.url),
                                         tuple(key_items))
-                response = self.handler.request(request=request.prepare(),
-                                                proxies=self.http.proxies,
-                                                timeout=timeout, **kwargs)
+                response = \
+                    self.handler.request(request=request.prepare(),
+                                         proxies=self.http.proxies,
+                                         timeout=timeout,
+                                         verify=self.http.validate_certs,
```

Comment:
Can you please use the name `validate_certs` rather than `verify`?

As for formatting, I suggest keeping `self.handler.request(` on the first line, and then indenting the arguments 6 or 8 spaces in from the beginning of the previous line.